instance/static/js/src/instance.js

```diff
@@ -122,7 +122,11 @@ app.controller("Index", ['$scope', 'Restangular', 'OpenCraftAPI', '$q',
             } else if(message.data.type === 'instance_log') {
                 if($scope.selected.instance && $scope.selected.instance.id === message.data.instance_id) {
                     $scope.$apply(function(){
-                        $scope.selected.instance.log_text += message.data.log_entry + '\n';
+                        if (message.data.log_entry.level == 'ERROR' || message.data.log_entry.level == 'CRITICAL') {
+                            $scope.selected.instance.errors.push(message.data.log_entry);
+                        } else {
+                            $scope.selected.instance.log_entries.push(message.data.log_entry);
```

Comment:
@chromano Don't we need to add errors to both `instance.errors` and `instance.log_entries`? Otherwise the errors will only show up in https://github.com/open-craft/opencraft/pull/13/files#diff-a8e64e20aeb5487bab76b72ae22e9aefR91 but not in https://github.com/open-craft/opencraft/pull/13/files#diff-a8e64e20aeb5487bab76b72ae22e9aefR92 ?